autoPyTorch/pipeline/components/training/trainer/__init__.py

```diff
@@ -381,6 +384,20 @@ def _fit(self, X: Dict[str, Any], y: Any = None, **kwargs: Any) -> 'TrainerChoic
 
         return self
 
+    def _get_train_label(self, X: Dict[str, Any]) -> List[int]:
+        """
+        Verifies and validates the labels from train split.
+        """
+        # Ensure that the split is not missing any class.
```

Comment:
What if we could not include some classes in the training split?
I think it can happen when we have few labels for some classes.

Reply:
I'd say then either the user should not use a weighted strategy or make a stratified split.

Reply:
Then we could raise a different kind of error message?